test/e2e/node_maintenance_test.go

```diff
@@ -459,30 +459,28 @@ func isTainted(node *corev1.Node) bool {
 }
 
 func hasValidLease(nodeName string, startTime time.Time) {
-	lease := &coordv1.Lease{}
-	err := Client.Get(context.TODO(), types.NamespacedName{Namespace: operatorNsName, Name: nodeName}, lease)
-	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "failed to get lease")
+	Eventually(func() {
```

Comment:
I think this is not doing what it is expected to do:
- I was wondering why it doesn't fail, because the lease name should have a lower case `node-` prefix
- For using `Expect` you need to use a `Gomega` object as func arg and do the `Expects` on that one, see https://onsi.github.io/gomega/#category-3-making-assertions-eminem-the-function-passed-into-codeeventuallycode